test/SparqlAntlrParserTestHelpers.h

```diff
@@ -132,39 +136,32 @@ constexpr const ad_utility::Last<Current, Others...>* unwrapVariant(
  * @param resultOfParseAndText Parsing result
  * @param matcher Matcher that must be fulfilled
  */
-void expectCompleteParse(const auto& resultOfParseAndText, auto&& matcher) {
+void expectCompleteParse(
+    const auto& resultOfParseAndText, auto&& matcher,
+    std::source_location l = std::source_location::current()) {
+  testing::ScopedTrace trace(l.file_name(), l.line(), "Source location");
```

Comment:
I would rather say `Actual location of test failure`, which seems more understandable:)